api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -175,7 +175,16 @@ func (in *IoChaosList) DeepCopyObject() runtime.Object {
 func (in *IoChaosSpec) DeepCopyInto(out *IoChaosSpec) {
```

Comment:
seem you can use https://github.com/jinzhu/copier to simplify the codes

Reply:
This is auto-generated, so I suggest making it be as it generated by scripts.